go.mod

```diff
@@ -7,14 +7,14 @@ go 1.13
 require (
 	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e // indirect
 	github.com/alecthomas/jsonschema v0.0.0-20190530235721-fd8d96416671
-	github.com/aws/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
+	github.com/aws/aws-sdk-go v1.25.48
```

Comment:
This is done by `go mod tidy`, however it doesn't matter much as we have the replace directive to weavework aws sdk (private)